js/google_ima.js

```diff
@@ -1752,6 +1759,16 @@ require("../html5-common/js/utils/utils.js");
             }
             break;
           case eventType.STARTED:
+            //Workaround of an issue on iOS where the IMA iframe is capturing clicks after an ad.
+            //We will show the iframe on receiving STARTED and hide it when receiving COMPLETE
+            if (OO.isIos) {
+              var IMAiframe = $("iframe[src^='http://imasdk.googleapis.com/']")[0];
```

Comment:
Is it possible to make this more specific by matching only iframes that are children of our player (in case there are other IMA iframes loaded on the page)?

Reply:
I think we should also leave out the protocol if possible in order to be able to match the https url.

Reply:
Do we have a reference to an element from our player? If we do then we could use `element.querySelector()` without having to use the class

Reply:
Ah, yep, we do have a reference to the container. Let me make this change.

Reply:
Added